src/Tools/ExternalAccess/Razor/RazorPredefinedCodeFixProviderNames.cs

```diff
@@ -0,0 +1,149 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Microsoft.CodeAnalysis.CodeFixes;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.Razor
+{
+    internal static class RazorPredefinedCodeFixProviderNames
```

Comment:
💡 Only include the ones Razor needs to use. Otherwise, any change to the internal name of _any_ code fix means we have to coordinate with another project.

Reply:
There is no issue with leaving Names in the Razor class that are no longer in the Roslyn class.